src/component/autocomplete.js

```diff
@@ -55,7 +55,9 @@ export class AutoCompleteCustomElement {
 
   //used to determine the string to be shown as option label
   @bindable label = result => {
-    return typeof result === 'object' ? result[this.attribute] : result;
+    return (typeof result === 'object' && result !== null && result[this.attribute])
+      ? result[this.attribute]
+      : '';
```

Comment:
These elaborate checks on `result` are not necessary if you need to check whether `result[this.attribute]` exists anyway. A simple truthyness check and short circuit should suffice:

```js
return (result && result[this.attribute]) || '';
```